AmplifyTools/AmplifyCLIiOS/Sources/AmplifyCLIiOS/Core/Support/String+Utils.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+extension String {
+    func homeDirectoryResolved() -> Self {
```

Comment:
This doesn't make sense as a String extension, for two reasons:

1) even though it might be operating on a String, it's not fundamentally related to Strings. Extensions should be related to the domain of the type itself; in this case, this method would make more sense extending FileManager or similar
2) That said, Apple obviously didn't listen to my opinion when they wrote the existing API [`NSString.expandingTildeInPath`](https://developer.apple.com/documentation/foundation/nsstring/1407716-expandingtildeinpath) :)

Reply:
we can definitely move that extension to `FileManager` (and file a rdar :p)